src/element/mod.rs

```diff
@@ -1494,7 +1494,7 @@ mod value_tests {
         let list: Element = ion_list![1, 2, 3].into();
         thread::scope(|_| {
             // Move `list` into this scope, demonstrating `Send`
-            let elements = vec![list];
+            let elements = [list];
```

Comment:
🗺️ Clippy noticed that this `vec!` (and the associated heap allocation) is not necessary.